lib/templates/fragments/header.php

```diff
@@ -2,41 +2,47 @@
 /**
  * Echo header fragments.
  *
- * @package Fragments\Header
+ * @package Beans\Framework\Templates\Fragments
+ *
+ * @since   1.0.0
  */
 
 beans_add_smart_action( 'beans_head', 'beans_head_meta', 0 );
 /**
  * Echo head meta.
  *
  * @since 1.0.0
+ *
+ * @return void
  */
 function beans_head_meta() {
-
 	?>
-	<meta charset="<?php esc_attr_e( get_bloginfo( 'charset' ) ); ?>" />
+	<meta charset="<?php esc_attr( get_bloginfo( 'charset' ) ); ?>" />
```

Comment:
Actually, we need to echo here.  This is not translated text.  Therefore, we can do this:

`echo esc_attr( get_bloginfo( 'charset' ) );`